flight/Libraries/math/misc_math.h

```diff
@@ -78,6 +78,100 @@ static inline bool IS_NOT_FINITE(float x) {
 	return (!isfinite(x));
 }
 
+/* Following functions from fastapprox https://code.google.com/p/fastapprox/
```

Comment:
Should this be a separate file? Arguments for: 1) the copyright makes more sense when it covers the entire file, 2) this isn't really a miscellaneous math function so much as it is an optimization for pre-existing functions.